### PR TITLE
Changes for easier inliner debugging

### PR DESCRIPTION
**Pull request by 9rnsr**
Separated from #5292:

Mainly for debugging.
1. Use `fd.loc` for pre-process expressions - declaring vret, vthis, and parameters.
2. Use `callLoc` for the expanded result expression.

### CODE REVIEW

File: src/inline.d [hunk 1]
```diff
@@ -35,8 +35,10 @@ import ddmd.tokens;
 import ddmd.visitor;
 
 private:
-enum LOG = false;
-enum CANINLINE_LOG = false;
+
+enum LOG = 0;
+enum CANINLINE_LOG = 0;
+enum EXPANDINLINE_LOG = 0;
```
Comment (WalterBright) [inline src/inline.d]: Why switch from bools to integers? I don't see any code that relies on them being anything but Boolean.

File: src/inline.d [hunk 2]
```diff
@@ -2137,9 +2139,7 @@ void expandInline(FuncDeclaration fd, FuncDeclaration parent, Expression eret,
         {
             auto vfrom = (*fd.parameters)[i];
 
-            auto arg = (*arguments)[i];
-            auto ei = new ExpInitializer(arg.loc, arg);
-
+            auto ei = new ExpInitializer(vfrom.loc, null);
```
Comment (WalterBright) [inline src/inline.d]: Why is `arg` dropped as the initializer?

File: src/inline.d (hunk 1 above)
Comment (9rnsr) [inline src/inline.d]: To switch the debug prints, rewriting `0` to `1`, or vice versa, is lower cost than rewiring between `true` and `false`.

File: src/inline.d (hunk 2 above)
Comment (9rnsr) [inline src/inline.d]: I cannot recall the exact reason... Will revert.

File: src/inline.d (hunk 1 above)
Comment (9rnsr) [inline src/inline.d]: However, I can personally use `0`/`1` during debugging, then can revert to `false` after my work. So, change back to use `false` for code readability.

Comment (WalterBright) [inline src/inline.d]: The trouble with using `0` is it implies that numbers other than `0` or `1` are possible, and so I spent time fruitlessly looking for that possibility. I recommend sticking with `true` and `false`.

Comment (9rnsr) [inline src/inline.d]: Sure, I reverted that.

File: src/inline.d (hunk 2 above)
Comment (9rnsr) [inline src/inline.d]: Done.

**Comment by WalterBright** [comment]
Auto-merge toggled on

**Comment by 9rnsr** [comment]
Thanks!

**Pull request merged**